### ISSUE TITLE
Custom saving path for medias saved

### ISSUE THREAD

**Issue by xerta555** [Snapcept/Snapcept#1]
Hello,
I'm using your `1.8.0` version (because there is no arm64 apk avalaible yet for the `10.34.1.0` beta version of SC for now).

For next versions of Snapcept, it could be possible to add an option to change the saving path please ?
I think this feature could be nice for this addon.

If you don't have the time i will understand no problem.
Thank you in advance.

**Comment by Stitchou** [comment]
Hi,  i'm on the 1.11.1 version

i down  snap's 35.0.0 no beta  here ==>https://apkpure.com/snapchat/com.snapchat.android/download/1741-APK?from=details%2Fversion


+1 for the custom saving path :)
I use 
xposed +XinternalSD for directly save whatsapp on SD but not working with Snapcept

EDIT : xposed +XinternalSD working after a reboot :) !!! snapcept directly save in SD Card

**Comment by georgeravenholm** [comment]
well that workaround worked for me aswell, thanks for posting the cool tip @Stitchou

**Comment by xerta555** [comment]
@Stitchou yeah i already know XInternalSD but it don't work on recent Android versions 😪

**Comment by Stitchou** [comment]
@xerta555 hi, what is your  android phone/version ? 

I'm on s7 edge with custom android 8 and all works fine for me 

 my rom( 2.0.3) from ==> https://forum.xda-developers.com/s7-edge/development/rom-galaxy-project-t3792787

@georgeravenholm :) your welcome

**Comment by xerta555** [comment]
@Stitchou seriously it works on your Oreo ROM ?? :astonished:
I try again this module 😂 

Edit: sorry i think u mean about Downloads2SD, wich isn't compatible with recent Android versions..

Sorry again.

**Comment by Stitchou** [comment]
Ahah no prob, try it then

I use XinternalSD for snapcept fb messenger and whatsapp 

Free internal memory 😊😊😊

**Comment by georgeravenholm** [comment]
well the workaround is good and all but i really think a setting in the module would be a better solution that shouldnt take too long to put in

perhaps also an option to generate a `.nomedia` file to hide the spam of photos from photo gallery for phones that support it
